docs/getting-started/installation/plesk.md

```diff
@@ -92,7 +92,7 @@ If you want to keep the folder in in a version controll like git, create a empty
 
 :::
 
-## Activate and configure node.js
+## 6. Activate and configure node.js
```

Comment:
The continuation of the enumeration does not make sense, because the enumeration started one hierarchy level below.